### ISSUE TITLE
Parallel Jim Culver (Laid to Rest) does not have the spirit deck's filter working correctly

### ISSUE THREAD

**Issue by lkstc112233** [Kamalisk/arkhamdb#634]
When creating a Parallel Jim Culver (with Laid to Rest as backside) deck, the spirit deck's requirements is not correctly enforced.

Example deck: https://arkhamdb.com/deck/view/3425307 I put all the allies in side deck.

Please let me know if I can provide any help.

**Comment by zzorba** [comment]
Side deck I meant to track upgrades and whatnot.

For ArkhamCards app I've been storing the spirit deck in the meta block (as a csv of card codes) in a field named "extra_deck".

The json data also has a set of fields to track the rules, but they'd need to be loaded by they database.

**Comment by lkstc112233** [comment]
I was about to say that - See attached link [1]. The data source uses `side_deck_options` and `side_deck_requirements`.

[1]: https://github.com/Kamalisk/arkhamdb-json-data/blob/1992451abae16389cb1e7e9d0a7a70f665dbc3c6/pack/parallel/ltr.json#L29-L35

**Comment by lkstc112233** [comment]
It seems to be the only card using that field, if you search for the field name.

**Comment by lkstc112233** [comment]
The field is not defined in the schema[1]. Maybe there's still chance to rename it?

[1]: https://github.com/Kamalisk/arkhamdb-json-data/blob/1992451abae16389cb1e7e9d0a7a70f665dbc3c6/schema/card_schema.json

**Comment by zzorba** [comment]
Yeah, renaming it on the card is an option still, it hasn't been used or hooked up anywhere yet.

Keeping the meta/extra_deck convention would be nice since some decks already use that.
